eqcorrscan/core/match_filter.py

```diff
@@ -4155,6 +4156,12 @@ def match_filter(template_names, template_list, st, threshold,
                       stream[0].stats.starttime.datetime.strftime('%Y%j')]),
             4, debug)
         if all_peaks[i]:
+            if len(all_peaks[i]) > 1000:
```

Comment:
This seems like quite an arbitrary threshold (not that there aren't other arbitrary thresholds in EQcorrscan), but it seems like something that should be kept outside of EQcorrscan and in a users own script. I have cases where I have thousands of detections in a single-stream and would rather not have this warning.